.github/workflows/master.yml

```diff
@@ -3,7 +3,7 @@ name: Continuous integration
 on:
   push:
     branches:
-      - master
+      - development
```

Comment:
@buckyroberts what branch do you think we should depend on? actually I do not see much value in separate `development` branch until we are in production (we could have merged to `master` until go live)

Reply:
We can deploy on pushes to `master`. If we are merging branches into `development` and then making more changes to `development` before everything is ready then we can keep `development` for now. If we are making all changes on the PR branches and only merging into `development` once everything is ready, then we can remove `development` for now and just set `master` as the default branch for the repo. Whatever is easiest for you @dmugtasimov

Reply:
@buckyroberts usually I do not accept breaking changes to either `development` or `master`. All fixes are done in feature branches. So in that sense `development` can be removed or abandoned. So I will make that change a little later.

@olegtropinin please, keep `development` here for now. I will migrate to `master` myself later